src/interfaces/models.ts

```diff
@@ -144,6 +144,11 @@ export interface IMLParsedNode {
 	readonly linkType?: DynamicContentTypes;
 
 	readonly attributes?: { [name: string]: string };
+
+	/**
+	 * If present, other nodes may refer to this node through the ID
+	 */
+	readonly elementId?: string;
```

Comment:
Okay, I see the point of keeping the ID contextless here (not naming it anchor), but perhaps in the component it should be named `anchor`.

Reply:
This is the only ref in the client:
```
	{node.elementId && (
		<a id={node.elementId}></a>
	)}
```
Where would you place the `anchor` thingie?